views/home.handlebars

```diff
@@ -1,186 +1,178 @@
-the bulk of the card's content.</p>
-</div>
-</div>
-</div>
-<div class="col-lg-3">
-<div class="card newsArticle">
-<div class="card-body">
-<a class="articleUrl" href="#">
-<h5 class="card-title articleTitle">Card title</h5>
-</a>
-<p class="card-text articleSummary">Some quick example text to build on the card
-title and make up
-the bulk of the card's content.</p>
-</div>
-</div>
-</div>
-<div class="col-lg-3">
-<div class="card newsArticle">
-<div class="card-body">
-<a class="articleUrl" href="#">
-<h5 class="card-title articleTitle">Card title</h5>
-</a>
-<p class="card-text articleSummary">Some quick example text to build on the card
-title and make up
-the bulk of the card's content.</p>
-</div>
-</div>
-</div>
-<div class="col-lg-3">
-<div class="card newsArticle">
-<div class="card-body">
-<a class="articleUrl" href="#">
-<h5 class="card-title articleTitle">Card title</h5>
-</a>
-<p class="card-text articleSummary">Some quick example text to build on the card
-title and make up
-the bulk of the card's content.</p>
-</div>
-</div>
-</div>
-</div>
-</div>
-</div>
-</div>
-<div class="col-lg-2 text-center">
-<div class="row">
-<div class="col-lg-12">
-<div class="watchList">
-<h5>Watch List</h5>
-<ul class="container text-center">
-</ul>
-</div>
-</div>
-</div>
-<div class="row my-4">
-<div class="col-lg-12">
-<button class="btn btn-warning btn-large">Add to watchList</button>
-</div>
-</div>
-</div>
-</div>
+            <div class="row">
+                <div class="col-lg-12" id="stockView">
+                    <br>
+                    <h6 id="priceDisplay" style="margin-left: 50px">$1,011.45</h6>
+                    <div class="row">
+                        <div class="col-lg-8">
+                            <canvas id="myChart"></canvas>
+                            <nav><button type="button" id="OneDayID"
+                                    onClick="updatingChart('', '/intraday-prices', 'OneDayID')"
+                                    class="badge badge-secondary" style="margin-left: 50px">1-Day</button>
+                                <button type="button" id="OneMonthID"
+                                    onClick="updatingChart('1m', '/chart/', 'OneMonthID')"
+                                    class="badge badge-secondary">1-Month</button>
+                                <button type="button" id="ThreeMonthID"
+                                    onClick="updatingChart('3m', '/chart/', 'ThreeMonthID')"
+                                    class="badge badge-secondary">3-Months</button>
+                                <button type="button" id="OneYearID"
+                                    onClick="updatingChart('1y', '/chart/', 'OneYearID')"
+                                    class="badge badge-secondary">1-Year</button>
+                                <button type="button" id="AllID" onClick="updatingChart('max', '/chart/', 'AllID')"
+                                    class="badge badge-secondary">ALL</button>
```

Comment:
These lines above, the updatingChart should only need period. then we parse period in the backend. for intra-day, best to pass something like "1d".
Let's do "5y" instead of "max" because I think max plots are always very slow.

Reply:
Done